diff --git a/node_modules/@hello-pangea/dnd/dist/dnd.esm.js b/node_modules/@hello-pangea/dnd/dist/dnd.esm.js
index df151d6..9f8af71 100644
--- a/node_modules/@hello-pangea/dnd/dist/dnd.esm.js
+++ b/node_modules/@hello-pangea/dnd/dist/dnd.esm.js
@@ -5303,7 +5303,7 @@ function useKeyboardSensor(api) {
 const idle = {
   type: 'IDLE'
 };
-const timeForLongPress = 120;
+const timeForLongPress = 0;
 const forcePressThreshold = 0.15;
 function getWindowBindings({
   cancel,
